packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx

```diff
@@ -130,21 +136,24 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <Drawer
+    <ModalPage
       isOpen
-      title={`Custom View: ${props.customView.defaultLabel}`}
       onClose={props.onClose}
       size={panelSize === 'small' ? 10 : 30}
+      title=""
```

Comment:
Shouldn't we provide a title here?

Reply:
The issue is that the title is required in the `ModalPage` component and I didn't want to change that but we don't want to display any title in this use case.

Reply:
The `title` of the `ModalPage` is only used for accessibility if I'm not mistaken, it's not a visible value.

Reply:
So are you suggesting we should make that property optional in the `ModalPage` component?

Reply:
No I'm suggesting to use it 😄